### ISSUE TITLE
An error occured during creating the thumbnail.

### ISSUE THREAD

**Issue by Vadimca** [GavickPro/GK-News-Show-Pro#41]
Hi guys.
My NSP plugin stopped working properly after upgrading to the newest WP version (4.2).
Now instead of images I get "An error occured during creating the thumbnail."
The site is http://voobshche-to.net/

**Comment by Vadimca** [comment]
Update...
The problem surprisingly fixed itself.. :|

**Comment by dziudek** [comment]
So the issue can be closed or not?

**Comment by Vadimca** [comment]
No, today the problem appeared again. That's weird because I did not change anything.
![error_screenshot](https://cloud.githubusercontent.com/assets/12109811/7335663/8c22db48-ec00-11e4-8ceb-9fe30cd319b5.jpg)

**Comment by dziudek** [comment]
In the data sources / wp / layout parts file please add before this line:

```php
if(!is_wp_error($img_editor)) {
```

Line:

```php
print_r($img_editor);
```

**Comment by Vadimca** [comment]
ok, the next time issue repeats I will add the line and get in touch. Thank you.

**Comment by grigoris19** [comment]
i have the same problem
Please look the screenshot
could you please to help me?
thank you
![now](https://cloud.githubusercontent.com/assets/14137005/9698716/1ec65f24-53cb-11e5-9231-ee1f3c18fd40.jpg)

**Comment by piotrkunicki** [comment]
Hi @grigoris19, the News theme uses another version of News Show Pro (built in theme files), do you have the latest version of the News? Please check also permissions of News/gavern/cache_nsp directory. If that won't help, please create new thread on the official News theme forum:
https://www.gavick.com/forums/news-wp
